code/game/machinery/computer/orders/order_computer/order_computer.dm

```diff
@@ -120,6 +120,10 @@ GLOBAL_LIST_EMPTY(order_console_products)
 				return
 			if(!purchase_items(used_id_card))
 				return
+			//So miners cant spam buy crates for a very low price
+			if(get_total_cost() < 200)
```

Comment:
might want to use the crate cost define here and on 125 instead of a hardcoded 200

Reply:
the default value is 200 and not 125, but im using the define since the value can change

Reply:
line 125*